aws/app/lambda/reliability/reliability.js

```diff
@@ -81,10 +82,9 @@ const getFormTemplate = async (formID) => {
         const response = JSON.parse(payload);
         const { records } = response.data;
         if (records?.length === 1 && records[0].formConfig.form) {
-          return {
-            formID,
-            ...records[0].formConfig.form,
-          };
+          const formTemplate = {formID, ...records[0].formConfig.form} 
```

Comment:
I think it could have been done this way but definitely not a major issue :)
```
return {
  formID,
  ...records[0].formConfig.form,
  securityAttribute: records[0].formConfig?.securityAttribute ?? ""
};
```